doc/getting-started.md

```diff
@@ -1,17 +1,35 @@
 # Getting started
 
-## Server side
+For installation instructions see [INSTALL.md](../INSTALL.md)
 
-To create an admin user and start soju, run these commands:
-
-    sojuctl create-user <soju username> -admin
-    soju -listen irc+insecure://127.0.0.1:6667
+## Migrating From ZNC
 
 If you're migrating from ZNC, a tool is available to import users, networks and
 channels from a ZNC config file:
 
     go run ./contrib/znc-import.go <znc config file>
 
+## Add your first user
+
+`sojuctl -config /etc/soju/config create-user <soju username> -admin` will add a new admin user for connecting to soju. You will be prompted for the soju user's password. The command must be run as a user with write access to the soju database.
```

Comment:
It was implicit before, soju and sojuctl cannot run at the same time because concurrent database access are not correctly implemented (can you confirm @emersion?)

This section should mention this in some way